pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -52,11 +56,50 @@ def __init__(self):
         self.delay = np.floor(self.latent_to_symptom_delay
                               / self.model_time_step)
 
+        # Infectiousness progression
+        # Instantiate parameters to be used in update infectiousness
+        infectious_profile = pe.Parameters.instance().infectiousness_prof
+        inf_prof_res = len(infectious_profile) - 1
```

Comment:
I'm confused what this is? If this is just an index for the last element in the array, it is more pythonic to use the index -1.

Reply:
It's the infectiousness profile resolution, it's a parameter equal to the length minus 1 of the infectious_profile array, used elsewhere than just for the infectious_profile array.